### PR TITLE
Removing internal menu styles hook

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Changes
- Removed checkmark custom styling hook used internally in menu

## Issues
- Updates #27058

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/efb2274e37c0237f3a10a4630b2696a9880f6398'>efb2274e37c0237f3a10a4630b2696a9880f6398</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/Avatar-mount_base.html">1337</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/Avatar-mount.html">1321</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/Button-mount_base.html">953</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/Button-mount.html">941</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/Field-mount_base.html">2038</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/Field-mount.html">2034</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/FluentProvider-mount_base.html">1657</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/FluentProvider-mount.html">1657</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/FluentProviderWithTheme-mount_base.html">629</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/FluentProviderWithTheme-mount.html">626</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">597</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">593</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">638</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">637</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/InfoButton-mount_base.html">571</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/InfoButton-mount.html">564</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/MakeStyles-mount_base.html">1908</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/MakeStyles-mount.html">1892</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/Persona-mount_base.html">3026</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/Persona-mount.html">3026</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/SpinButton-mount_base.html">2452</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27085/perf-test-react-components/SpinButton-mount.html">2455</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Review by ling1726 — APPROVED** [review]

File: change/@fluentui-react-shared-contexts-2a428543-5453-4c0a-846b-196b5734538d.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Removed checkmark custom styles hook",
```
Comment (khmakoto) [inline change/@fluentui-react-shared-contexts-2a428543-5453-4c0a-846b-196b5734538d.json:3]:
```suggestion
  "comment": "fix: Removing internal checkmark custom styles hook",
```

**Review by khmakoto — APPROVED** [review]

**Pull request merged**